state-chain/custom-rpc/src/lib.rs

```diff
@@ -420,29 +420,6 @@ pub struct RpcSwapOutputV2 {
 	pub network_fee: RpcFee,
 	pub ingress_fee: RpcFee,
 	pub egress_fee: RpcFee,
-}
-
-impl From<RpcSwapOutputV3> for RpcSwapOutputV2 {
-	fn from(swap_output: RpcSwapOutputV3) -> Self {
-		Self {
-			intermediary: swap_output.intermediary.map(Into::into),
-			output: swap_output.output,
-			network_fee: swap_output.network_fee,
-			ingress_fee: swap_output.ingress_fee,
-			egress_fee: swap_output.egress_fee,
-		}
-	}
-}
-
-#[derive(Serialize, Deserialize, Clone)]
-pub struct RpcSwapOutputV3 {
```

Comment:
I removed this type because it just adds a new field, which is considered a non-breaking change in the json api.